lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebarNavigation.test.tsx

```diff
@@ -4,20 +4,26 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
 import NxGlobalSidebarNavigation, { NxGlobalSidebarNavigationProps as Props } from '../NxGlobalSidebarNavigation';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 describe('NxGlobalSidebarNavigation', function() {
   const minimalProps: Props = {},
-      getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxGlobalSidebarNavigation, minimalProps);
+      renderEl = rtlRenderElement<Props>(NxGlobalSidebarNavigation, minimalProps);
 
-  it('renders a <nav> with nx-global-sidebar__navigation class and those passed as props', function() {
-    expect(getShallowComponent({ className: 'class-A' })).toMatchSelector('nav.nx-global-sidebar__navigation.class-A');
+  it('passes additional classes specified as props to <nav>', function() {
```

Comment:
`<nav>` isn't being tested here.  Check for the `navigation` role.

Reply:
> `<nav>` isn't being tested here. Check for the `navigation` role.

Added here: 29a5e37f3f19c8e851e36ca3c1c6c4b0d2e42bc3

Reply:
> Added here: https://github.com/sonatype/sonatype-react-shared-components/commit/29a5e37f3f19c8e851e36ca3c1c6c4b0d2e42bc3

That's good but you didn't remove the mentions of `<nav>` from the other test names (which are not in fact testing for the `nav` tagName)

Reply:
> > Added here: [29a5e37](https://github.com/sonatype/sonatype-react-shared-components/commit/29a5e37f3f19c8e851e36ca3c1c6c4b0d2e42bc3)
> 
> That's good but you didn't remove the mentions of `<nav>` from the other test names (which are not in fact testing for the `nav` tagName)

Removed here: dffc31526a06b0b950b8b23f9c3f3316cf973987